packages/devtools_app/lib/src/screens/performance/tabbed_performance_view.dart

```diff
@@ -64,11 +65,15 @@ class _TabbedPerformanceViewState extends State<TabbedPerformanceView>
   @override
   Widget build(BuildContext context) {
     late Widget frameAnalysisView;
+    if (controller.data == null) {
+      return const SizedBox();
+    }
     final selectedFrame = _selectedFlutterFrame;
     if (selectedFrame != null) {
       frameAnalysisView = FlutterFrameAnalysisView(
         frameAnalysis: selectedFrame.frameAnalysis,
         enhanceTracingController: controller.enhanceTracingController,
+        rebuildCountModel: controller.data!.rebuildCountModel,
```

Comment:
The 
```
if (controller.data == null) {
      return const SizedBox();
    }
    ```
    Code makes it seem like `controller.data` may be null sometimes.
    It seems like `!` here may end up crashing for us sometime as a result.

Reply:
cleaned up code so no `!` were required. that helped fix one case that might have NPE